watertap/examples/flowsheets/RO_with_energy_recovery/RO_with_energy_recovery.py

```diff
@@ -160,6 +160,7 @@ def build(erd_type=ERDtype.pressure_exchanger):
     m.fs.costing.add_specific_electrical_carbon_intensity(
         m.fs.product.properties[0].flow_vol
     )
+    m.fs.objective = Objective(expr=m.fs.costing.LCOW)
```

Comment:
Why put this here instead of where it originally was under `optimize_setup`? If it is only for the sake of checking the objective exists in tests, that could happen after the optimize_setup function is called in the test.